pkg/apis/v2beta2/config_conversion.go

```diff
@@ -0,0 +1,340 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2beta2
+
+import (
+	"github.com/kubesphere/notification-manager/pkg/apis/v2beta1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this Config to the Hub version (v2beta2).
```

Comment:
to the Hub version (v2beta2)
=>
to the Hub version (v2beta1) ？
because dst is v2beta1 : `dst := dstRaw.(*v2beta1.Config)`